File: praktikum_6/PRAK602-2410817210004-Rahmad.c
#include <stdio.h>
int main(){
    int batas;
    scanf("%d",&batas);
    int bilangan[batas];
    int hasilkali[batas];
    for(int i=0;i<batas;i++){
        scanf("%d",&bilangan[i]);
    }
    for(int i =0;i<batas;i++){
    hasilkali[i]=bilangan[i]*(i+1);
    }
    for(int i =0;i<batas;i++){
    printf("%d ",hasilkali[i]);
    }
}

File: praktikum_6/PRAK602-2410817210004-Rahmad.py
batas=int(input(""))
bilangan=[]
hasilkali=[]

bilangan = list(map(int, input("").split()))
for i in range(batas):
    hasilkali.append(bilangan[i]*(i+1))
for i in range(batas):
    print(f"{hasilkali[i]}",end=" ")